streams/src/test/java/org/apache/kafka/streams/integration/AbstractResetIntegrationTest.java

```diff
@@ -612,6 +612,30 @@ private void cleanGlobal(final Properties sslConfig,
         Assert.assertEquals(0, exitCode);
     }
 
+    void assertInputTopicDoesNotExist() throws Exception {
+
+        final Properties streamsConfiguration = prepareTest();
+        final List<String> parameterList = new ArrayList<>(
+                Arrays.asList("--application-id", APP_ID + testNo,
+                        "--bootstrap-servers", bootstrapServers,
+                        "--input-topics", INPUT_TOPIC));
+
+        final String[] parameters = parameterList.toArray(new String[parameterList.size()]);
+        final Properties cleanUpConfig = new Properties();
+        cleanUpConfig.put(ConsumerConfig.HEARTBEAT_INTERVAL_MS_CONFIG, 100);
+        cleanUpConfig.put(ConsumerConfig.SESSION_TIMEOUT_MS_CONFIG, "" + CLEANUP_CONSUMER_TIMEOUT);
+
+        // RUN
+        KafkaStreams streams = new KafkaStreams(setupTopologyWithoutIntermediateUserTopic(), streamsConfiguration);
+        streams.start();
+
+        final int exitCode = new StreamsResetter().run(parameters, cleanUpConfig);
```

Comment:
I assume this fails, because KS is still running. This is a valid test, but does test if `StreamsResetter` fails for non-existing topic.

Reply:
Let me change the input-topics in the parameterlist. Then i believe this should fail for sure.